visualizer/index.html

```diff
@@ -203,7 +242,7 @@
     #bottomSection {
       display: flex;
       flex: 1;
-      flex-direction: column;
+      overflow-x: auto;
```

Comment:
This still produces double horizontal scroll bars for me. I'm using the ES5 grammar, with the following input: `var x = "asdlfkjasdlfkjasdf";`. The parse tree is big enough to trigger scrolling -- on master, I just see a single horizontal scroll bar, but on this branch, I see two.

Currently, the way the scrolling works is this: `#bottomSection` and `#visualizerBody` do not scroll -- only `#parseResults` does. I did that because I wanted the expanded input to always be visible, and not scroll out of view.    But the problem with that is that we want to be able to scroll the expanded input horizontally, while still keeping it visible when the parse tree is scrolled vertically. So I see what you're trying to do here.

What could work is if we make `#bottomSection` scroll _only_ horizontally (like you've done here), and I change `#parseResults` to scroll only vertically. That will eliminate the double scroll bar problem, and should give us the behaviour that we want.